ghdocs/Onboarding/Building.md

```diff
@@ -0,0 +1,65 @@
+# [sp-client](../../../README.md) › Building
+
+This document describes how to setup your development environment to build the **office-ui-fabric-react** repository and contribute changes.
```

Comment:
Contributing.md maybe?

I see contributions.md in there but it doesn't seem useful.